aeolus/global.h

```diff
@@ -61,11 +59,10 @@ enum // GLOBAL LIMITS
     NKEYBD = 6,
     NGROUP = 8,
     NNOTES = 61,
-    NBANK  = 32,
-    NPRES  = 32
```

Comment:
these were clears for alignment, which results in better readability should should stay